docs/lcdproc-user/drivers/futaba.docbook

```diff
@@ -0,0 +1,140 @@
+  TOSD-5711BB USB VFD display. This as commonly used on Elonex Artisan,
+  Fujitsu Scaleo E and FIC Spectra Media Centre PCs.</para>
+
+  <sect2 id="futaba-displays">
+    <title>Displays</title>
+
+    <para>The Futaba TOSD-5711BB is a VFD based USB 2.0 display, very much in
+    the style of VHS Recorders/Players from the 1990s. It contains a single
+    line, 7 Character 14-segment display and multiple icons to show the media
+    type, codec, etc; current state of the PC; Volume knob and indicator. Each
+    segment will display one character (Letters can only be in Capitals), and
+    has a '<literal>:</literal>' (colon) and '.' (dot) between each character.
+    There is no contrast control or back-light. The volume knob is lit using a
+    blue LED and spins eternally. An image of the VFD circuit board and Volume
+    knob can be seen <ulink
+    url="http://images.esellerpro.com/2131/I/772/87/DCP_1700.JPG"><uri>here</uri></ulink>.</para>
+
+    <para><inlinemediaobject>
+        <imageobject>
+          <imagedata fileref="http://images.esellerpro.com/2131/I/772/87/DCP_1700.JPG"/>
+        </imageobject>
+      </inlinemediaobject></para>
+
+    <para>The IR attached is a standard MCE eHome remote which does not need
+    any new drivers to work with lirc. It also work with
+    <application>scripts.xbmc.lcdproc</application> on
+    <application>Kodi</application>.</para>
+
+    <para>It comes pre-installed on Elonex Artisan, Fujitsu Scaleo E and FIC
+    Spectra Media Centre PCs <ulink
+    url="http://www.fic.com.tw/spectraviiv/drivers/SPECTRA%20ID1%20Viiv%20User%20Manual.pdf">FIC
+    Spectra Manual</ulink>.</para>
+
+    <note>
+      <itemizedlist>
+        <listitem>
+          <para>If the display crashes on start-up, this diff will solve the
+          problem: <ulink
+          url="https://github.com/ajw107/LibreELEC.tv/blob/master/packages/linux/patches/default/linux-061-add-elonex-artisan-vfd-support.patch">Linux
+          Diff</ulink>. I have submitted this patch for inclusion into the
+          standard Linux Kernel.</para>
+        </listitem>
+
+        <listitem>
+          <para>The patched version of
+          <application>scripts.xbmc.lcproc</application> can be found <ulink
+          url="https://github.com/ajw107/script.xbmc.lcdproc">here</ulink>,
+          this should be added upstream soon too.</para>
+        </listitem>
+      </itemizedlist>
+    </note>
+  </sect2>
+
+  <sect2 id="futaba-requirements">
+    <title>Requirements</title>
+
+    <para>The driver is based on the <ulink
+    url="http://libusb.info/"><filename>libusb</filename></ulink> USB library
+    (Versions 1.0 and 0.1), which should make it work with Linux, the
+    different BSB variants as well as Darwin/MacOS X.</para>
+
+    <note>
+      <para>When using a <filename>libusb</filename> based driver like
+      <code>futaba</code>, <application>LCDd</application> needs to be started
+      as root.</para>
+    </note>
```

Comment:
Will dropping priviledges after initialization (the usual LCDd way) work or has LCDd run as root all the time?